frontend/packages/noobaa-storage-plugin/src/components/bucket-class/create-bc.tsx

```diff
@@ -0,0 +1,145 @@
+import * as React from 'react';
+import { Wizard } from '@patternfly/react-core';
+import './create-bc.scss';
+import { referenceForModel, apiVersionForModel, k8sCreate } from '@console/internal/module/k8s';
+import * as _ from 'lodash';
+import { history } from '@console/internal/components/utils/router';
+import { match } from 'react-router';
+import { ClusterServiceVersionModel } from '@console/operator-lifecycle-manager/src/models';
+import { NooBaaBackingStoreModel, NooBaaBucketClassModel } from '../../models';
+import { GeneralPage } from './wizard-pages/general-page';
+import { PlacementPolicyPage } from './wizard-pages/placement-policy-page';
+import { BackingStorePageConnected } from './wizard-pages/backingstore-page';
+import { ReviewPage } from './wizard-pages/review-page';
+import { reducer, initialState } from './state';
+
+enum CreateStepsBC {
+  GENERAL = 'GENERAL',
+  PLACEMENT = 'PLACEMENT',
+  BACKINGSTORE = 'BACKINGSTORE',
+  REVIEW = 'REVIEW',
+}
+
+const CreateBC: React.FC<CreateBCProps> = (props) => {
+  const [state, dispatch] = React.useReducer(reducer, initialState);
+  const ns = _.get(props.match, 'params.ns');
+  const appName = _.get(props.match, 'params.appName');
+
+  // TODO(Bipul):Add support for secondary tier
+  const finalStep = () => {
+    const payload = {
+      apiVersion: apiVersionForModel(NooBaaBucketClassModel),
+      kind: NooBaaBucketClassModel.kind,
+      metadata: {
+        name: state.bucketClassName,
+        namespace: ns,
+      },
+      spec: {
+        placementPolicy: {
+          tiers: [
+            {
+              tier: {
+                mirrors: [
+                  {
+                    mirror: {
+                      [state.tier1Policy]: [...state.tier1BackingStore],
+                    },
+                  },
+                ],
+              },
+            },
+          ],
+        },
+      },
+    };
```

Comment:
Its better to move the payload to some constant and replace the fields which needs to be updated.